tests/test_plugins.py

```diff
@@ -12,15 +12,16 @@ def setUp(self):
             title="home",
             template="page.html",
             language=self.language,
+            published=True,
```

Comment:
This will throw a warning, it doesn't actually do anything any more, it was kept because it's part of the api.

Reply:
published can be removed altogether